entente/test_rigid_transform.py

```diff
@@ -89,6 +89,10 @@ def test_rigid_rotation_single_point():
 
 
 @pytest.mark.skipif(env_flag("CI") is True, reason="failing in CI when numpy>=1.19.3")
+# This test currently has two valid answers.  On numpy 1.19.3 and higher it returns a different
+# but still accurate value.
+# https://github.com/lace/entente/issues/195
+# TODO: re-write test to be less symmetrical.
 def test_rigid_rotation_with_reflection():
     a = Box(origin=np.array([0.0, 0.0, 0.0]), size=np.array([1.0, 1.0, 1.0])).v
```

Comment:
Just a hunch, I wonder if we changed size to `np.array([1.0, 2.0, 5.0])` if it would make the answer unique.

(I suspect there are three correct answers currently.)